CHANGELOG.md

```diff
@@ -1,3 +1,19 @@
+# 🚛 [4.16.0](https://github.com/ionic-team/stencil/compare/v4.15.0...v4.16.0) (2024-04-15)
+
+
+### Bug Fixes
+
+* **cli:** fix a bug in CLI argument parsing ([#5646](https://github.com/ionic-team/stencil/issues/5646)) ([1fdea63](https://github.com/ionic-team/stencil/commit/1fdea63acfa5a9c1081111d7d79e826a127ef3eb))
```

Comment:
```suggestion
* **cli:** fix a bug in CLI argument parsing ([#5646](https://github.com/ionic-team/stencil/issues/5646)) ([1fdea63](https://github.com/ionic-team/stencil/commit/1fdea63acfa5a9c1081111d7d79e826a127ef3eb)), closes [#5640](https://github.com/ionic-team/stencil/issues/5640)
```

Reply:
I just chatted with Alice about this, this change alone does not resolve #5640 entirely

Reply:
@rwaskiewicz Also for vis ^

Reply:
Gotcha, in this case we can skip it or replace `closes` with just `refs`.

Reply:
__Updated:__ tweaked the code suggestion to `refs`